Snowflake.Data.Tests/IntegrationTests/SFConnectionIT.cs

```diff
@@ -1540,6 +1540,57 @@ public void TestInvalidProxySettingFromConnectionString()
             }
         }
 
+        [Test]
+        [TestCase("*")]
+        [TestCase("*{0}*")]
+        [TestCase("^*{0}*$")]
```

Comment:
maybe add a testcase for ^ without $ and vice versa as well regarding the change in HttpUtil

Reply:
Added requested cases